File: UserServer/src/main/java/cc/nobrain/dev/userserver/domain/notice/service/dto/NoticeRes.java
package cc.nobrain.dev.userserver.domain.notice.service.dto;

import cc.nobrain.dev.userserver.domain.base.dto.FileDto;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@Getter
@NoArgsConstructor
public class NoticeRes extends NoticeDto {
    private List<FileDto> attachments;
}

File: UserServer/src/main/kotlin/cc/nobrain/dev/userserver/domain/notice/service/dto/NoticeRes.kt
package cc.nobrain.dev.userserver.domain.notice.service.dto

import cc.nobrain.dev.userserver.domain.base.dto.BaseDto
import cc.nobrain.dev.userserver.domain.base.dto.FileDto
import java.io.Serializable

data class NoticeRes(
    var attachments: List<FileDto> = emptyList()
) : NoticeDto(
    id = 0,
    title = "",
    content = "",
    sticky = false
), Serializable